x-pack/qa/sql/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
 
   // JDBC testing dependencies
   compile project(path: xpackModule('sql:jdbc'), configuration: 'nodeps')
+  compile project(path: xpackModule('sql:sql-proto'))
```

Comment:
What makes this required?

Reply:
It's not needed anymore since we depend on `sql-action` (it's redundant) so I removed it.